targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_SDK_15_0/components/libraries/util/app_util_platform.c

```diff
@@ -85,7 +92,11 @@ void app_util_critical_region_exit(uint8_t nested)
     /* return value can be safely ignored */
     (void) sd_nvic_critical_region_exit(nested);
 #else
-    app_util_enable_irq();
+    /** Mbed modification
+	 *  Retarget nRF SDK to use Mbed critical section API
+	 */
+	//app_util_enable_irq();
+	core_util_critical_section_exit();
```

Comment:
Is this aligned properly ? The line 93 is 4 spaces left, these `else` lines should also be

Reply:
Fixed in new commit.